packages/web-twig/README.md

```diff
@@ -78,6 +78,21 @@ For available components see the [components directory](https://github.com/lmc-e
 if you want to extend these components, an example guide is [here](./docs/extendComponents.md).
 if you want to contribute, read the guide [here](./CONTRIBUTING.md).
 
+## Additional Attributes
+
+All components accept additional attributes that are passed down to the root element of the component.
+This is useful for adding custom event handlers, accessibility attributes, or other attributes that
+are not supported by the component API.
+
+Supported attributes are:
+
+- `on*` (eg. `onclick`)
```

Comment:
Please explain how the `on*` attributes/props are passed to the native element (`<input>`, `<button>`, …).